.github/workflows/checkpatch.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Tools that can save round-trips to github and a lot of time:
+#
+# yamllint -f parsable pull_request.yml
+# pip3 install ruamel.yaml.cmd
+# yaml merge-expand pull_request.yml exp.yml &&
+#    diff -w -u pull_request.yml exp.yml
+#
+# github.com also has a powerful web editor that can be used without
+# committing.
+
+# This is the name of this workflow and should technically be called
+# something like "Main Workflow" but the place where most people see
+# this name is the Checks window next to other, non-github checks.
+name: checkpatch
+
+# yamllint disable-line rule:truthy
+on: pull_request
+
+jobs:
+  checkpatch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set up env for checkpath
+        run: |
+          sudo apt-get update
```

Comment:
Is this necessary?

Reply:
why not to always use the latest apt resources? I can remove if you think it is not needed.